src/python/pants/engine/README.md

```diff
@@ -136,27 +145,24 @@ def concat(files):
   yield ConcattedFiles(''.join(fc.content for fc in file_content_list))
 ```
 
-This `@rule` declares that: "for any Subject for which we can compute `Files`, we can also compute
-`ConcattedFiles`". Each yielded `Get` request results in FileContent for a different File Subject
-from the Files list.
+This `@rule` declares that: "for any `Params` for which we can compute `Files`, we can also compute
+`ConcattedFiles`". Each yielded `Get` request results in FileContent for a different File `Param`
+from the Files list. And, happily, all of these requests will proceed in parallel.
```

Comment:
I was torn on how to respond to this, because while the distinction is meaningful, I'm not sure how useful it is. But I agree that "will proceed in parallel" is only true if you have multiple cores, so "can proceed in parallel" would be more accurate.